client/Components/matchCard.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { render } from 'react-dom';
+import { connect } from 'react-redux';
+import { toJS } from 'immutable';
+import request from 'superagent';
+
+var injectTapEventPlugin = require("react-tap-event-plugin");
+injectTapEventPlugin();
+
+const Card = require('material-ui/lib/card/card');
+const CardActions = require('material-ui/lib/card/card-actions');
+const CardExpandable = require('material-ui/lib/card/card-expandable');
+const CardHeader = require('material-ui/lib/card/card-header');
+const CardMedia = require('material-ui/lib/card/card-media');
+const CardText = require('material-ui/lib/card/card-text');
+const CardTitle = require('material-ui/lib/card/card-title');
+const RaisedButton = require('material-ui/lib/raised-button');
+
+export const MatchCard = React.createClass({
+  componentWillMount() {
+    request.get('/users/' + this.props.match.get('challengee'))
+      .end((err, res) => {
+        if (err) {
+          console.error(err)
+        } else {
+          this.challengee = res.body.data.username;
+          window.location.hash="#/matches";
```

Comment:
is this just a way to re-render the matches page? seems odd to have to do it in a componentWillMount. N.B. a better way to re-render is to say this.render() :)